Octokit.Tests.Integration/Reactive/ObservableTeamsClientTests.cs

```diff
@@ -10,13 +10,13 @@ public class ObservableTeamsClientTests
 {
     public class TheGetMembersMethod
     {
-        readonly Team team;
+        readonly Team _team;
```

Comment:
This is a coding style thing, which is cool and all but there's no other changes around here.

Can we revert this to simplify the diff?